release/charts/arguments.go

```diff
@@ -0,0 +1,132 @@
+package charts
```

Comment:
What's the context for the name arguments? If it's related to the command(s) being ran, that's an implementation leak and should be renamed to represent the business logic and now how the logic is used.

Reply:
Implemented a fix from what I could at understand at:
`"changing naming strategy from arguments.go -> chart_management.go"`
Commit.